.github/workflows/main.yml

```diff
@@ -7,12 +7,44 @@ on:
     types:
       - published
 jobs:
+  verify-docs:
+    name: Verify docs
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [12.x]
+    steps:
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.x'
```

Comment:
Do we need this (given that I was able to remove it in the other workflow?

Reply:
Yes as we actually do run mike locally in this build which builds and deploys the docs using the insiders mkdocs version